### PR TITLE
Added wrist PID calibration / cleanup

### PR DESCRIPTION
**Pull request by Jnesselr**
The branch is `break_glass` as in "break glass in case of emergency". to try and make sure we can have a robot in time, I'm helping the students out with writing code. There's a bit of cleanup in here, but one big thing was calibrating the wrist position. I also tested the flippers run by servos and found that they won't work as we intended.

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Shooter.java [hunk 1]
```diff
@@ -34,14 +38,11 @@ public class Shooter extends SubsystemBase {
   private final String SMART_DASHBOARD_POSITION = "Shooter Motor Position";
   private final String SMART_DASHBOARD_TARGET_POSITION = "Shooter Motor Target Position";
 
-  private final CANcoder armPositionEncoder = new CANcoder(18);
-  private double targetArmPosition = 0;
+  private final CANcoder armPositionEncoder = new CANcoder(18, Constants.OperatorConstants.canivoreSerial);
+  private double targetArmPosition = 40;
```
Comment (awfulcarpet) [inline src/main/java/frc/robot/subsystems/Shooter.java:42]: possibly comment here what 40 means in the context of the robot.

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -116,7 +120,7 @@ private void configureBindings() {
 
     if (Utils.isSimulation()) {
       drivetrain.seedFieldRelative(new Pose2d(new Translation2d(), Rotation2d.fromDegrees(90)));
-    
+
     drivetrain.registerTelemetry(telemetry::telemeterize);
```
Comment (awfulcarpet) [inline src/main/java/frc/robot/RobotContainer.java:124]: indentation?

**Review by awfulcarpet — APPROVED** [review]
just tiny things that don't necessarily need to be fixed

**Review by centiwati — APPROVED** [review]
LGTM 😜

**Review by GucciBanana11 — APPROVED** [review]
LGTM 😜 

In all seriousness, I couldn't find anything worth noting

**Pull request merged**